src/Nethermind/Nethermind.BeaconNode.Peering/MothraNetworkPeering.cs

```diff
@@ -38,5 +47,35 @@ public Task PublishBeaconBlockAsync(SignedBeaconBlock signedBlock)
 
             return Task.CompletedTask;
         }
+
+        public Task RequestBlocksAsync(string peerId, Root peerHeadRoot, Slot finalizedSlot, Slot peerHeadSlot)
+        {
+            // TODO: BeaconBlocksByRange is separate work item... just log that we got here.
+
+            LogDebug.RpcSend(_logger, RpcDirection.Request, nameof(MethodUtf8.BeaconBlocksByRange), peerId, 0, null);
+            //throw new NotImplementedException();
+
+            return Task.CompletedTask;
+        }
+
+        public Task SendStatusAsync(string peerId, RpcDirection rpcDirection, PeeringStatus peeringStatus)
+        {
+            byte[] peerUtf8 = Encoding.UTF8.GetBytes(peerId);
+            Span<byte> encoded = new byte[Ssz.Ssz.PeeringStatusLength];
+            Ssz.Ssz.Encode(encoded, peeringStatus);
```

Comment:
is status sent once per connection or more often? if the latter then maybe we should cache the byte[] form of peering status

Reply:
It is sent once at the beginning (it is a "hello" message).  The spec says you may want to periodically, after a long time, resend, to see if things have changed (e.g. peer has switched to a different branch), but that would be a recalculated status anyway, with different final block + head.